src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,92 @@
+.stopwatch {
+  width: $size;
+  height: $size;
+  border-radius: 25%;
+
+  display: flex;
+  justify-content: center;
+
+  border: 3px solid $color-second-minute-border;
+  box-shadow: 0 18px 40px rgba(0,0,0,0.15);
+
+  position: relative;
+
+  &::after {
+    content: "";
+    top: 15%;
+
+    width: 5%;
+    height: 5%;
+    border-radius: 50%;
+
+    background-color: $color-milisecond-and-dot;
+
+    position: relative;
+  }
```

Comment:
i can't figure out what it's for, but it looks cool

Reply:
@ruslanpashkov 
It's like a mark point to see the midday/midnight or start position :)

![image](https://user-images.githubusercontent.com/81112084/234294622-48a0b70e-3f45-456c-9874-f5a2d8e41785.png)

Reply:
pretty confusing